src/installers/helm.ts

```diff
@@ -162,8 +162,18 @@ error: E_COMMAND_FAILED`)
       tlsFlag = `-f ${destDir}values/tls.yaml`
     }
 
-    let command = `helm upgrade --install che --force --namespace ${flags.chenamespace} --set global.ingressDomain=${flags.domain} --set cheImage=${flags.cheimage} --set global.cheWorkspacesNamespace=${flags.chenamespace} ${multiUserFlag} ${tlsFlag} ${destDir}`
-    await execa.shell(command, { timeout: execTimeout })
+    await execa('helm',
+				['upgrade',
+				'--install', 'che',
+				'--force',
+				'--namespace', flags.chenamespace,
+				'--set', 'global.ingressDomain=${flags.domain}',
+				'--set', 'cheImage=${flags.cheimage}',
+				'--set', 'global.cheWorkspacesNamespace=${flags.chenamespace}',
+				'${multiUserFlag}',
```

Comment:
multiUserFlag and tlsFlag are in fact 2 arguments.
i think multiUserFlag is activated by default, if true, it seems that it is working even written this way, withotu the ned to split the arguments.